arch/arm/core/cortex_m/cmse/arm_core_cmse.c

```diff
@@ -70,3 +70,70 @@ int arm_cmse_addr_range_readwrite_ok(u32_t addr, u32_t size, int force_npriv)
 	return arm_cmse_addr_range_read_write_ok(addr, size, force_npriv, 1);
 }
 
+#if defined(CONFIG_ARM_SECURE_FIRMWARE)
+
+int arm_cmse_mpu_nonsecure_region_get(u32_t addr, u8_t *p_region)
```

Comment:
Shouldn't we have similar function for checking secure region?

Reply:
We do already.  We have arm_cmse_mpu_region_get on line 10 which works for secure and for non-secure.

Reply:
Ok